src/proposals/20250306/SparkEthereum_20250306.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { IERC20 } from 'forge-std/interfaces/IERC20.sol';
+
+import { IAaveV3ConfigEngine as IEngine } from '../../interfaces/IAaveV3ConfigEngine.sol';
+import { ICapAutomator }                  from '../../interfaces/ICapAutomator.sol';
+
+import { SparkPayloadEthereum, Rates, EngineFlags } from "../../SparkPayloadEthereum.sol";
+
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+/**
+ * @title  March 06, 2025 Spark Ethereum Proposal
+ * @notice SparkLend: Onboard LBTC, tBTC
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/march-6-2025-proposed-changes-to-spark-for-upcoming-spell/26036
+ * Vote:   TODO
+ */
+contract SparkEthereum_20250306 is SparkPayloadEthereum {
+
+    address internal constant AGGOR_BTCUSD_ORACLE = 0x4219aA1A99f3fe90C2ACB97fCbc1204f6485B537;
+
+    constructor() {
+        // PAYLOAD_BASE = 0x;
+    }
+
+    function _preExecute() internal override {
+        LISTING_ENGINE.POOL_CONFIGURATOR().setEModeCategory({
+            categoryId:           3,
+            ltv:                  85_00,
+            liquidationThreshold: 90_00,
+            liquidationBonus:     102_00,
+            oracle:               AGGOR_BTCUSD_ORACLE,
```

Comment:
We do not use an oracle override for e-mode. This should be `address(0)` same as the other e-modes.

Reply:
Updated